test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.pte.live;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import de.schildbach.pte.NetherlandsProvider;
+import de.schildbach.pte.dto.Point;
+
+/**
+ * @author Guus Hoekman
+ */
+public class NetherlandsProviderLiveTest extends AbstractNavitiaProviderLiveTest {
+    public NetherlandsProviderLiveTest() {
+        super(new NetherlandsProvider(secretProperty("navitia.authorization")));
+    }
+
+    @Test
+    public void nearbyStationsAddress() throws Exception {
+        nearbyStationsAddress(52080225, 4323195);
+    }
+
+    @Test
+    public void nearbyStationsAddress2() throws Exception {
+        nearbyStationsAddress(52069885, 4320715);
+    }
+
+    @Test
+    public void nearbyStationsStation() throws Exception {
+        nearbyStationsStation("stop_point:OAM:2323791");
+    }
+
+    @Test
+    public void nearbyStationsPoi() throws Exception {
+        nearbyStationsPoi("poi:n457396814");
+    }
+
+    @Test
+    public void nearbyStationsAny() throws Exception {
+        nearbyStationsAny(52080225, 4323195);
+    }
+
+    @Test
+    public void nearbyStationsInvalidStation() throws Exception {
+        nearbyStationsInvalidStation("stop_point:OAM:23237945");
+    }
+
+    @Test
+    public void queryDeparturesEquivsFalse() throws Exception {
+        queryDeparturesEquivsFalse("stop_point:OAM:2323791");
```

Comment:
Here I used the same as nearbyStationsStation, like in the FranceNorthEastProvider file